demos/inpage/hardhat/package.json

```diff
@@ -4,9 +4,10 @@
   "main": "index.js",
   "private": true,
   "dependencies": {
-    "@account-abstraction/contracts": "^0.6.0",
-    "@openzeppelin/contracts": "^4.9.3",
-    "hardhat": "^2.17.0"
+    "account-abstraction": "github:eth-infinitism/account-abstraction#0.6.0",
+    "hardhat": "^2.17.3",
+    "openzeppelin-contracts": "github:openzeppelin/openzeppelin-contracts#v4.9.3",
+    "safe-contracts": "github:safe-global/safe-contracts#v1.4.1"
```

Comment:
Yeah there seems to be a small benefit - it allows you to use the same import paths as you do by default when using foundry. This makes compatibility easier because solidity doesn't have package management (as far as I know) so when your dependency has a dependency you need to install that yourself.